backend/src/schema/types/type/User.gql

```diff
@@ -157,6 +159,7 @@ type Query {
 }
 
 type Mutation {
+  acceptTermsAndCondition(version: String!): Boolean
```

Comment:
```suggestion
```
We can use `UpdateUser` for this because we're changing properties on the user object.